cloud-service-providers/azure/azureml/cli/azureml_files/deployment.yml

```diff
@@ -3,7 +3,7 @@ name: deployment_name_placeholder
 endpoint_name: endpoint_name_placeholder
 environment: 
   name: image_name_placeholder-env
-  image: acr_registry_placeholder.azurecr.io/image_name_placeholder:v1
```

Comment:
Can you remove "latest" and mention "tag" instead to make it generalized.

Reply:
When using the v1 it was crashing

Reply:
I tried the way it was and the deployment was throwing errors as it couldn't find the image, the errors went away with latest